tritonbench/utils/triton_op.py

```diff
@@ -39,6 +39,8 @@
     tqdm = None
 
 logger = logging.getLogger(__name__)
+# TODO: remove this once we have a better way to handle backward benchmarking
+torch._functorch.config.donated_buffer = False
```

Comment:
This seems an overkill to me. Can we disable it only in backward and forward_backward?

Reply:
fixed in https://github.com/pytorch-labs/tritonbench/pull/104/commits/20b138d024ab39697642fb6bf9db8c9d969ebd5e